### PR TITLE
User / Record Book Seeds

### PR DESCRIPTION
**Pull request by Tybot204**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -8,36 +8,91 @@
...
+users << { discord_name: 'Sloth#6910' }         unless User.exists? discord_name: 'Sloth#6910'
+User.create! users
+
+puts 'done'
+
+print 'Seeding Teams...'
+
+teams = []
+teams << { name: 'Alpha Team' } unless Team.exists? name: 'Alpha Team'
+teams << { name: 'Beta Team' }  unless Team.exists? name: 'Beta Team'
```
Comment (NicksPatties) [inline db/seeds.rb]: I thought it was Bravo Team...

File: db/seeds.rb [hunk 2]
```diff
@@ -8,36 +8,91 @@
...
+
+record_books = []
+unless RecordBook.exists? name: 'Season 2'
+  record_books << {
+    name:            'Season 1',
+    published:       true,
+    start_time:      2.weeks.ago,
+    end_time:        2.weeks.from_now,
+    rush_start_time: 1.week.from_now,
+    rush_end_time:   2.weeks.from_now
```
Comment (NicksPatties) [inline db/seeds.rb]: What's the difference between `start_time` and `rush_start_time`?

File: db/seeds.rb (hunk 1 above)
Comment (Tybot204) [reply db/seeds.rb]: Shhhh my bad.

File: db/seeds.rb (hunk 2 above)
Comment (Tybot204) [reply db/seeds.rb]: This is down to how we've planned record books working. Each month we have a new record book, where only clan members can participate. The last week or so new applicants get to join in with a set of new challenges. That period is called "Rush Week" and has it's own time frame.

**Pull request merged**